test/allreduce_torchrun.py

```diff
@@ -45,7 +46,7 @@ def _mp_fn_xla_backend():
   print(
       'master_port:{}, master_addr:{}, rank:{}, local_rank:{}, size:{}'.format(
           os.environ['MASTER_PORT'], os.environ['MASTER_ADDR'],
-          os.environ['RANK'], os.environ['LOCAL_RANK'],
+          os.environ['RANK'], os.environ[xenv.XLA_LOCAL_RANK],
```

Comment:
nit: how do you feel about `PTXLA_LOCAL_RANK` just to differentiate ourselves from tensorflow xla?

Reply:
XLA should be fine, many of our env var started with XLA.